pandas/tests/frame/test_arithmetic.py

```diff
@@ -1147,6 +1147,33 @@ def test_arithmetic_midx_cols_different_dtypes_different_order(self):
         expected = DataFrame([[-1, 1], [-1, 1]], columns=midx)
         tm.assert_frame_equal(result, expected)
 
+    @pytest.mark.xfail(reason="NaT op NaT results in datetime instead of timedelta")
+    def test_arithmetic_datetime_df_series(self):
+        # GH#59529
+        df_datetime = DataFrame([[1, 2], [3, 4]]).astype("datetime64[ns]")
+        ser_datetime = Series([5, 6, 7]).astype("datetime64[ns]")
+        result = df_datetime - ser_datetime
+        expected = DataFrame(
+            [
+                [pd.Timedelta(-4), pd.Timedelta(-4), pd.Timedelta(pd.NaT)],
+                [pd.Timedelta(-2), pd.Timedelta(-2), pd.Timedelta(pd.NaT)],
+            ]
+        )
+        tm.assert_frame_equal(result, expected)
+
+    @pytest.mark.xfail(reason="NaT op NaT results in datetime instead of timedelta")
+    def test_arithmetic_timestamp_timedelta(self):
```

Comment:
IIUC these tests are mainly about the alignment behavior, so they belong in something like tests.frame.test_arithmetic

Reply:
The tests are currently in pandas/tests/frame/test_arithmetic.py. Should they be moved somewhere else?